package.json

```diff
@@ -10,10 +10,13 @@
   },
   "dependencies": {
     "blockstack": "^19.3.0",
+    "bp-vuejs-dropdown": "^2.1.1",
     "browser-lang": "^0.1.0",
     "grid.css": "^0.6.1",
     "marked": "^0.7.0",
+    "node-sass": "^4.12.0",
     "nuxt": "^2.9.2",
+    "sass-loader": "^8.0.0",
```

Comment:
we don't use sass

Reply:
sorry i was trying different component that used it and i forgot